addons/website_event/views/event_templates_list.xml

```diff
@@ -99,9 +99,9 @@
                                 <div t-attf-class="o_wevent_offcanvas_cat_#{category.name} accordion-collapse collapse">
                                     <div class="accordion-body pt-0">
                                         <ul class="list-group list-group-flush">
-                                            <li t-if="category.is_published and category.tag_ids and any(tag.color for tag in category.tag_ids)" class="list-group-item">
-                                                <t t-foreach="category.tag_ids" t-as="tag">
-                                                    <a t-if="tag.color"
+                                            <t t-if="category.is_published and category.tag_ids and any(tag.color for tag in category.tag_ids)" t-foreach="category.tag_ids" t-as="tag">
+                                                <li t-if="tag.color" class="list-group-item border-0 px-0">
+                                                    <a t-att-href="'/event?%s' % keep_query('*', tags=str((search_tags - tag).ids if tag in search_tags else (tag | search_tags).ids))"
```

Comment:
Seems like this is a duplicate of the following line?

Reply:
I don't understand, can you explain?